MUSHclient/worlds/plugins/aard_channels_fiendish.xml

```diff
@@ -190,6 +190,18 @@ if window_title == "" then
    window_title = nil
 end
 
+function clearTab(tab)
+   if tab == nil then
+      tab = 1
```

Comment:
you don't want to default to the current tab?

Reply:
Good point. Changed.